vegafusion-python/tests/specs/tooltip_value_labels.json

```diff
@@ -0,0 +1,277 @@
+{
+    "layer": [
+        {
+            "layer": [
+                {
+                    "layer": [
+                        {
+                            "mark": {
+                                "clip": true,
+                                "type": "bar",
+                                "color": "#2266D3",
+                                "tooltip": false
+                            },
+                            "encoding": {
+                                "x": {
+                                    "sort": "descending",
+                                    "type": "nominal",
+                                    "field": "product_category",
+                                    "scale": {
+                                        "type": "linear"
```

Comment:
Invalid scale type for nominal field

Using "type": "linear" scale for a nominal product_category field is incorrect. Linear scales are for continuous quantitative data.
```
                                    "scale": {
-                                        "type": "linear"
+                                        "type": "band"
                                    }
```